webapp/src/components/Metrics/DeltaComputationBar.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { screen } from "@testing-library/react";
+import { renderWithTheme } from "mocks/utils";
+import DeltaComputationBar from "./DeltaComputationBar";
+
+const renderDeltaComputationBar = (
+  value: number,
+  formattedValue: string,
+  width: number
+) =>
+  renderWithTheme(
+    <DeltaComputationBar
+      formattedValue={formattedValue}
+      value={value}
+      width={width}
+    />
+  );
+
+describe("DeltaComputationBar", () => {
+  it("should display the bar and the formatter value with expected bgcolor, width and position if the value is greater than 0", () => {
+    renderDeltaComputationBar(45.5, "45.5", 45.5);
+    expect(screen.getByTestId("delta-computation-bar")).toHaveStyle(
+      "width:45.5%;background-color:rgba(99, 204, 255, 0.8);left:50%"
+    );
+    expect(screen.getByTestId("delta-computation-value")).toHaveTextContent(
+      "+45.5"
+    );
+    expect(screen.getByTestId("delta-computation-value")).toHaveStyle(
+      "color:rgb(0, 109, 179);right:55%"
+    );
+  });
+
+  it("should display the bar with expected bgcolor, width and position if the value is less than 0", () => {
+    renderDeltaComputationBar(-45.5, "45.5", 45.5);
+    expect(screen.getByTestId("delta-computation-bar")).toHaveStyle(
+      "width:45.5%;background-color:rgba(37, 36, 54, 0.6);right:50%"
+    );
+    expect(screen.getByTestId("delta-computation-value")).toHaveTextContent(
+      "45.5"
+    );
```

Comment:
Isn't the point of this test to make sure negative values work?
```suggestion
    renderDeltaComputationBar(-45.5, "-45.5", 45.5);
    expect(screen.getByTestId("delta-computation-bar")).toHaveStyle(
      "width:45.5%;background-color:rgba(37, 36, 54, 0.6);right:50%"
    );
    expect(screen.getByTestId("delta-computation-value")).toHaveTextContent(
      "-45.5"
    );
```

Reply:
In the [code](https://github.com/ServiceNow/azimuth/blob/main/webapp/src/components/Metrics/DeltaComputationBar.tsx#L64) and as per the design, we display just the value with no sign if its negative.

Reply:
Wrong. Everywhere we use a `DeltaComputationBar`, the `formattedValue` is passed with a minus sign already.
![image](https://user-images.githubusercontent.com/8386369/220946757-bfab34a3-1a08-41d6-8e26-ef84bdd7ef61.png)

Reply:
Oh yeah. Sorry. I missed the details here. I found the issue as well. I am passing +45 to formattedValue, so my test did not accept -45. I corrected the passing arguments.
Another thing I noticed today is elem.textContent captures the content correctly, then using "toHaveTextContent." I am trying to understand why it does that. My. next commit will resolve this test case. Thanks for your input :)

Reply:
Hey @JosephMarinier  FYI, I replaced "toHaveTextContent" with element.textContent where ever I used it. I could also find some test cases failing after using element.textContent and fixed the test cases too.

Reply:
Perfect! Thank you!
I think it was clear in my above suggestion that you needed to add the minus sign in those two places. Please read carefully.

Reply:
I checked your suggestion [here](https://github.com/ServiceNow/azimuth/pull/420#discussion_r1115204314) right after I found what's causing the issue and added the fix and saw you are suggesting the same 👍 Thank you for the suggestion here